lib/dream_html.mli

```diff
@@ -664,6 +664,9 @@ module Hx : sig
   val vals : _ string_attr
   val ws_connect : _ string_attr
   val ws_send : attr
+  
+  val hs: _ string_attr
```

Comment:
Could you reorder this into alphabetical position in the `Hx` module? Also, could you add a little more documentation about what it is because at first glance it may not be obvious that `hs` renders as the `_` attribute.